src/config.ml

```diff
@@ -1692,6 +1717,9 @@ let a_not_implemented =
       string "explicit-exit-notify";
       string "tun-ipv6";
       string "ifconfig-ipv6";
+      string "max-clients";
+      string "setenv";
+      string "push";
```

Comment:
It was useful for testing to ignore `push` directives, but eventually we may want to support `push` at least to some degree.
```suggestion
```